appengine/websockets/app.yaml

```diff
@@ -1,4 +1,18 @@
-# [START appengine_websockets_yaml]
+# Copyright 2018-2025 Google LLC
```

Comment:
we don't do this. It should just be Copyright 2018 as the starting year. We don't want to have to update this each time the year changes.